learningresources/api.py

```diff
@@ -301,3 +302,20 @@ def import_static_assets(course, path):
                 name = join(root, name).replace(path + sep, '', 1)
                 django_file.name = name
                 create_static_asset(course.id, django_file)
+
+
+def update_xanalytics(filename):
+    """
+    Update xanalytics fields for a LearningResource.
+    Args:
+        filename (string): filename of JSON file from xanalytics
+    """
+    with open(filename) as raw:
+        data = json.load(raw)
```

Comment:
If we're getting rid of the file, then this will already be getting a valid dict. But yeah, there should be exception handling and logging in the part where we retrieve and parse the JSON from the API.